fitbenchmarking/fitbenchmarking
# Performance profiler

#### Description of Work

Fixes #418 by producing a performance profile for the acc and runtime tests.

#### Testing Instructions

1. Run fitbenchmarking
2. Check performance profiles show up correctly on acc and runtime results
3. Check nothing shows for the other tables

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
